frontend/src/navbar/NavbarMenu.tsx

```diff
@@ -154,6 +154,11 @@ function allStartItems(toggleExpansion: (item: string) => void): NavbarItem[] {
                     icon: <MilitaryTech />,
                     href: '/tournaments/open-classical',
                 },
+                {
+                    name: 'Round Robin',
+                    icon: <MilitaryTech />,
+                    href: '/tournaments/round-robin',
```

Comment:
Why is this change included?

Reply:
feel free to remove this in the grad branch, I'm already working on this in the tournament UI

Reply:
The beta branch got mixed in when resolving merge conflicts.